tests/Browser/BookingsPageTest.php

```diff
@@ -100,4 +100,44 @@ public function testClickOnReviewBooking()
         });
     }
 
+    public function testUserCanViewABookingRequestStatus()
+    {
+        $bookings = BookingRequest::factory()
+            ->count(1)
+            ->hasReservations(random_int(1, 3))
+            ->create(["status" => BookingRequest::PENDING])->first();
+
+        $this->browse(function (Browser $browser) use ($bookings) {
+            $browser->loginAs(User::first());
+            $browser->visit('/bookings')
+                ->clickLink("View", 'button')
+                ->pause(5000);
```

Comment:
```suggestion
                ->pause(250);
```
5 seconds is a longggggg time. If every test had a 5 second delay, CI would take hours! :p 

Try with a small delay and work your way up